aws/iac/master-compliance-test.json

```diff
@@ -19255,6 +19255,74 @@
                         }
                     ],
                     "resourceTypes": []
+                },
+                {
+                    "masterTestId": "PR-AWS-CFR-AMF-001",
+                    "masterSnapshotId": [
+                        "CFR_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(amplify.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AWS-CFR-AMF-001",
+                            "eval": "data.rule.amplify_basic_auth",
+                            "message": "data.rule.amplify_basic_auth_err",
+                            "remediationDescription": "Make sure you are following the Cloudformation template format presented <a href='https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-amplify-app.html' target='_blank'>here</a>",
+                            "remediationFunction": "PR_AWS_CFR_AMF_001.py"
+                        }
+                    ],
+                    "severity": "High",
+                    "title": "Ensure AWS amplify has basic auth enabled.",
+                    "description": "To enhance the security enable basic auth for Amplify. It Enables basic authorization for the autocreated branch. So for this, user requires to enter base64-encode authorization credentials.",
+                    "tags": [
+                        {
+                            "cloud": "git",
+                            "compliance": [
+                                "CIS"
```

Comment:
@jaiminswan CIS the only complinace? 
can we add Best Practices as well?

Reply:
@farchide yes, let me add it